backend/src/backend/main.py

```diff
@@ -68,6 +70,11 @@ async def test(timestamp: int):
 async def add_book(book_payload: BookAddSchema):
     """API endpoint to receive Book addition requests and add to database"""
 
+    ident = get_ident_from_name(book_payload.metadata["Name"])
+    if ident:
```

Comment:
I'd prefer this function to fail rather than return an empty string. Ideally, common use cases should return a ValueError with a descriptive message.